app/src/pages/inside/common/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -87,6 +87,8 @@ export class Hamburger extends Component {
     window.location.href = URLS.exportLaunch(this.props.projectId, this.props.launch.id, type);
   };
 
+  isInProgress = () => this.props.launch.status === 'IN_PROGRESS';
```

Comment:
Please use the corresponding constant from `app/src/common/constants/launchStatuses.js`